apps/src/music/player/ToneJSPlayer.ts

```diff
@@ -223,7 +223,7 @@ class ToneJSPlayer {
     await this.startContextIfNeeded();
     events.forEach(({notes, playbackPosition}) => {
       const offsetSeconds = Transport.toSeconds(
-        this.playbackTimeToTransportTime(playbackPosition)
+        this.playbackTimeToTransportTime(playbackPosition - 1)
```

Comment:
Mind leaving a quick comment for why we have to do this?

Reply:
Good call.  Added in https://github.com/code-dot-org/code-dot-org/pull/64912.